public/locales/en-US/send.ftl

```diff
@@ -152,8 +152,8 @@ addPasswordMessage = Protect with password
 addPasswordLabel = Password:
 copyUrlLabel = Copy and share this link:
 passwordReminder = don't forget the password too
-signInPromoText = Sign In/Up!
-signInExplanation = It's free and gives you many more Send options
+signInPromoText = Sign In/Up
+signInExplanation = It's free and you can send bigger files.
```

Comment:
Need new string IDs for both messages
https://developer.mozilla.org/en-US/docs/Making_String_Changes

Reply:
these strings haven't hit master yet